src/components/molecules/WorldNav/WorldNavLabelMap.ts

```diff
@@ -3,7 +3,7 @@ import { WorldNavTab } from "types/world";
 export const WorldNavLabelMap: Readonly<
   Record<WorldNavTab, string>
 > = Object.freeze({
-  [WorldNavTab.start]: "Start",
+  [WorldNavTab.general]: "General settings",
```

Comment:
Must it have `settings` here and not after `Advanced` or even after `Entrance`? Is this a comment for the issue, not just this PR?

Reply:
It's a valid question. One for product team to resolve at a later date.

Reply:
Ping @whoaitsaimz